### ISSUE TITLE
Time Range button: Incorrect role as 'radio button' is defined instead of tab items for 'Fiscal year' and 'Time zone' control time range pop-up. 

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#86658]
### What happened?

Incorrect role as 'radio button' is defined instead of tab items for 'Fiscal year' control time range pop-up. 
The narrator announces it here as 'Time zone, radio button 1 of 2 selected' etc. 

https://github.com/grafana/grafana/assets/93735775/4ac176de-d6e7-4ab9-9f2d-737e0f8c2eab

### What did you expect to happen?

The correct role as tab items be defined for all these two controls Narrator should announce as 'Time zone, 1 of 2, selected' etc. 

### Did this work before?

No

### How do we reproduce it?

1. Open the URL [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1) in edge Dev browser.
2. ‘Grafana Play Home’ page will open. 
3. Navigate to ‘Time range’ button & activate it. 
4. Navigate to ‘Change time settings’ button & activate it. 
5. Navigate to the 'Fiscal year' and 'Time zone' controls & observe the screen reader narration. 

### Environment (with versions)?

Grafana: v11.1.0-69372 (51da96d94e)
OS: 24H2 (OS Build 26100.1) 
Browser: Edge version 125.0.2506.0 (Official build) dev (64-bit)

### Grafana platform?

None
